File: back/Models/Review.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MatchJob.Models
{
    public class Review
{
    public Guid Id { get; set; } = Guid.NewGuid();

    public Guid ReviewerId { get; set; }
    public User Reviewer { get; set; } = null!;

    public Guid ProfessionalProfileId { get; set; }
    public ProfessionalProfile ProfessionalProfile { get; set; } = null!;

    public int Rating { get; set; }
    public string? Comment { get; set; }

    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}
}

File: mobile/src/types/review.ts
export interface Review {
  Id: string;
  ReviewerId: string;
  ReviewerName: string;
  ProfessionalProfileId: string;
  Rating: number;
  Comment: string | null;
  CreatedAt: string;
}

export interface CreateReviewRequest {
  Rating: number;
  Comment?: string;
}
